wishlist/wishlist.html

```diff
@@ -0,0 +1,122 @@
+            <li class="nav-love-icon pl-l">
+                <a class="nav-loveicon-a" href=""><i class="far fa-heart f-l"></i></a>
+                <span class="nav-number f-s br-s">5</span>
+            </li>
+            <li class="pl-l">
+                <div class="badge">
+                    <a class="nav-bag-icon" href=""><i class="fa fa-shopping-cart badge-icon relative f-l"></i></a>
+                    <span class="badge-greenNumber br-s">10</span>
+                </div>
+            </li>
+        </ul>
+    </nav>
+    <!--main body-->
+    <h1 class="whislist-heading mt-l f-xl font-xl">My Wishlist</h1>
+    <div class="whislist-product flex flex-center mb-l  mt-l">
+        <div class="cards_withBadge">
+            <div class="header">
+                <image class="card-image_one" src="../assets/shopping.webp" alt="controler-image">
+                    <span class="card-icon p-xs br-s"><i class="far fa-heart"></i></span>
+            </div>
+            <div class="body  flex flex-center flex-column">
+                <div class="p-xss"> wireless controller</div>
+                <div class="f-m font-xs">रु5690</div>
+            </div>
+            <div class="footer p-xs">
+                <p>Add to cart</p>
+            </div>
+        </div>
+        <div class="cards_withBadge">
+            <div class="header">
+                <image class="card-image_one" src="../assets/shopping.webp" alt="controler-image">
+                    <span class="card-icon p-xs br-s"><i class="far fa-heart"></i></span>
+            </div>
+            <div class="body  flex flex-center flex-column">
+                <div class="p-xss"> wireless controller</div>
+                <div class="f-m font-xs">रु5690</div>
+            </div>
+            <div class="footer p-xs">
+                <p>Add to cart</p>
+            </div>
+        </div>
+        <div class="cards_withBadge">
+            <div class="header">
+                <image class="card-image_one" src="../assets/shopping.webp" alt="controler-image">
+                    <span class="card-icon p-xs br-s"><i class="far fa-heart"></i></span>
+            </div>
+            <div class="body  flex flex-center flex-column">
+                <div class="p-xss"> wireless controller</div>
+                <div class="f-m font-xs">रु5690</div>
+            </div>
+            <div class="footer p-xs">
+                <p>Add to cart</p>
+            </div>
+        </div>
+        <div class="cards_withBadge">
+            <div class="header">
+                <image class="card-image_one" src="../assets/shopping.webp" alt="controler-image">
+                    <span class="card-icon p-xs br-s"><i class="far fa-heart"></i></span>
+            </div>
+            <div class="body  flex flex-center flex-column">
+                <div class="p-xss"> wireless controller</div>
+                <div class="f-m font-xs">रु5690</div>
+            </div>
+            <div class="footer p-xs">
```

Comment:
you can also use span, small element tag instead of div

Reply:
there a reason for using it